src/ExperienceForm.js

```diff
@@ -0,0 +1,85 @@
+// src/ExperienceForm.js
+import React, { useState } from 'react';
+import './ExperienceForm.css';
+
+function ExperienceForm() {
+  const [formData, setFormData] = useState({
+    title: '',
+    company: '',
+    start_date: '',
+    end_date: '',
+    description: '',
+    logo: ''
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData({
+      ...formData,
+      [name]: value
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    fetch('/resume/experience', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(formData)
+    })
+      .then(response => response.json())
+      .then(data => {
+        console.log('Success:', data);
```

Comment:
We could show an alert box to the user here and the other places where the user needs to know about the successful response